README.md

```diff
@@ -59,6 +59,10 @@ If you want Geminabox to carry on providing gems when rubygems.org is unavailabl
 
     Geminabox.allow_remote_failure = true
 
+By default, local gems always take precedence over upstream gems—even if the upstream gems are newer!—to prevent ["dependency confusion" supply-chain attacks](https://medium.com/@alex.birsan/dependency-confusion-4a5d60fec610) wherein an attacker pushes a remote gem with the same name as your local gem, causing you to inadvertently install a malicious gem when you `gem install` or `bundle install`.
```

Comment:
This isn't the default behaviour. This is what happens if the strategy is set to `:local_gems_take_precedence_over_remote_gems`

Reply:
I'd also suggest that this section gets a heading that matches the error message. So add (before the paragraph current starting "By default, local gems always ... ):
```
### Merge Strategy
```
So that when a user gets the error "Merge strategy must be ..." they can find the relevant section of the README easily.

Reply:
> This isn't the default behaviour. This is what happens if the strategy is set to :local_gems_take_precedence_over_remote_gems

This PR changes the default strategy to `:local_gems_take_precedence_over_remote_gems`.

Reply:
> So add (before the paragraph currently starting "By default, local gems always ... ):

I'll leave communication and release strategy to the software maintainers. I've reverted changes to the README.

Reply:
> > This isn't the default behaviour. This is what happens if the strategy is set to :local_gems_take_precedence_over_remote_gems
> 
> This PR changes the default strategy to `:local_gems_take_precedence_over_remote_gems`.

Yes you're right - I missed the setting of the default in lib/geminabox.rb.